pkg/network/dhcp/masquerade_test.go

```diff
@@ -95,11 +111,29 @@ var _ = Describe("Masquerade DHCP configurator", func() {
 		BeforeEach(func() {
 			mockHandler.EXPECT().LinkByName(ifaceName).Return(iface, nil)
 		})
-		It("Should return the dhcp configuration", func() {
-			config, err := generator.Generate()
-			Expect(err).ToNot(HaveOccurred())
-			Expect(*config).To(Equal(generateExpectedConfig(vmiSpecNetwork, nil, mtu, ifaceName)))
+
+		When("IPv6 is enabled", func() {
```

Comment:
maybe worth to rename generateExpectedConfig to generateExpectedConfigIPv6Enabled ?

Reply:
Thanks. I merged the two functions into one, so it should be OK now.

Reply:
> maybe worth to rename generateExpectedConfig to generateExpectedConfigIPv6Enabled ?

Done.
https://github.com/kubevirt/kubevirt/compare/4f0b45fd16ec19310894f412f2e650f9c0c91e4e..909f4bdc5045b3bef8bdc1b1f7df7fc0a0bc7ba3